composer.json

```diff
@@ -7,45 +7,44 @@
         "php": "^7.2",
         "ext-json": "*",
         "ext-pdo": "*",
-        "joomla/application": "^2.0@dev",
-        "joomla/console": "^2.0@dev",
-        "joomla/controller": "^2.0@dev",
-        "joomla/database": "^2.0@dev",
-        "joomla/di": "^2.0@dev",
-        "joomla/event": "^2.0@dev",
-        "joomla/filesystem": "^2.0@dev",
-        "joomla/filter": "^2.0@dev",
+        "fig/link-util": "^1.1",
+        "joomla/application": "^2.0",
+        "joomla/console": "^2.0",
+        "joomla/controller": "^2.0",
+        "joomla/database": "^2.0",
+        "joomla/di": "^2.0",
+        "joomla/event": "^2.0",
+        "joomla/filesystem": "^2.0",
+        "joomla/filter": "^2.0",
         "joomla/github": "^2.0@dev",
-        "joomla/http": "^2.0@dev",
-        "joomla/input": "^2.0@dev",
-        "joomla/model": "^2.0@dev",
-        "joomla/preload": "^2.0@dev",
-        "joomla/registry": "^2.0@dev",
-        "joomla/renderer": "^2.0@dev",
-        "joomla/router": "^2.0@dev",
-        "joomla/string": "^2.0@dev",
-        "joomla/uri": "^2.0@dev",
-        "joomla/utilities": "^2.0@dev",
-        "joomla/view": "^2.0@dev",
-        "fig/link-util": "^1.0",
-        "monolog/monolog": "^1.25.1",
+        "joomla/http": "^2.0",
+        "joomla/input": "^2.0",
+        "joomla/model": "^2.0",
+        "joomla/preload": "^2.0",
+        "joomla/registry": "^2.0",
+        "joomla/renderer": "^2.0",
+        "joomla/router": "^2.0",
+        "joomla/string": "^2.0",
+        "joomla/uri": "^2.0",
+        "joomla/utilities": "^2.0",
+        "joomla/view": "^2.0",
+        "laminas/laminas-diactoros": "^2.3",
```

Comment:
Should be ^2.11.1

Reply:
Unfortunately 2.11.1 is not compatible with out minimum PHP requirements. Yes, we have to check that anyway and update accordingly.